### PR TITLE
Fix (#9761) Korean locales

### LINKED ISSUE #9761: Korean localization resource looks broken for v1.11
- Gitea version (or commit ref): v1.11
- Git version: 1.8.3.1
- Operating system: CentOS 7
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes (provide example URL)
  - [ ] No
  - [ ] Not relevant
- Log gist:

## Description
Korean resource looks broken. I guess localization files for Korean defiled.
...

## Screenshots
image

image

### PR DESCRIPTION
**Pull request by 6543**
close #9761

### CODE REVIEW

**Review by sapk — APPROVED** [review]

**Review by zeripath — APPROVED** [review]

File: options/locale/locale_ko-KR.ini [hunk 1]
```diff
@@ -14,18 +14,18 @@ page=페이지
...
-create_new=ㅊㄱㄷㅁㅅㄷ...
-user_profile_and_more=ㅖ개랴ㅣㄷ 뭉 ㄴㄷㅅ샤ㅜㅎㄴ...
+create_new=생성하기
+user_profile_and_more=프로파일 및 설정
 signed_in_as=다음 사용자로 로그인됨
-enable_javascript=쏘ㅑㄴ ㅈ듀냣ㄷ 재간 ㅠㄷㅅㅅㄷㄱ 쟈소 ㅓㅁㅍㅁㄴㅊ갸ㅔㅅ.
+enable_javascript=이 웹사이트는 자바스크립트 활성화가 필요합니다.
 
 username=사용자명
 email=뜨먀ㅣ ㅁㅇㅇㄱㄷㄴㄴ
```
Comment (bagasme) [inline options/locale/locale_ko-KR.ini]: This string still broken

Comment (6543) [reply options/locale/locale_ko-KR.ini]: Then we have to wait for anothwr crowdin update

Comment (6543) [reply options/locale/locale_ko-KR.ini]: @bagasme updated

**Comment by 6543** [comment]
Would say ready to merge 🚀 ?

**Pull request merged**